### ISSUE TITLE
test.pypi.org Account recovery request

### ISSUE THREAD

**Issue by danking** [pypi/support#4708]
### PyPI Username

danking

### Reason for request

Lost access to email address (dking@broadinstitute.org).

NB: this is for TestPyPI not real PyPI (I have a separate ticket for real PyPI: https://github.com/pypi/support/issues/4658)

I'd like to switch to daniel.zidan.king@gmail.com

### Recovery Codes

- [X] I never generated or have lost access to the recovery codes for my account

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct

### Acknowledgement

- [X] I understand that it may take a significant amount of time to process my account recovery request.

**Comment by Thespi-Brain** [comment]
Hi @danking,

Thanks for sending over your preferred email in this issue. But, since we have an account verification process as due diligence before we can update your email, please contact [support@pypi.org](mailto:support@pypi.org) with your preferred email address and reference this link (https://github.com/pypi/support/issues/4708). We will proceed from there. Thanks!

**Comment by Thespi-Brain** [comment]
Hi @danking,

I am following up regarding your request. Are you still having an issue regarding your account's email address? Let us know! I will keep this issue open for another week unless I hear from you earlier. After that, if you are still having trouble, feel free to create a new request. Thanks!

**Comment by Thespi-Brain** [comment]
Closing due to non response.
